lisp_ease/src/queries/utils.lisp

```diff
@@ -0,0 +1,289 @@
+(in-package :le)
```

Comment:
Organize utils better. Maybe put them into categories and divide your file accordingly. Use 
``;;;;;;;;;;;;;;;;;;;;;;;;;;;;; SOME CATEGORY ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;``
for splitting the file. This is sometimes used in the rest of CRAM, so it will be uniform.
E.g., bullet utils -- one category, transform util, query lisp utils etc.